### PR TITLE
Adding in more time

### PR DESCRIPTION
**Pull request by apeltzer**
Add new default of 4h for default jobs. 

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@
   an AWS S3 bucket as the `--outdir`.
 * Fix workflow.onComplete() message when finishing pipeline
 * Update URL for joining the nf-core slack to https://nf-co.re/join/slack
+* [Increased default time limit](https://github.com/nf-core/tools/issues/370) to 4h
```
Comment (apeltzer) [inline CHANGELOG.md]:
```suggestion
* [Increased default time limit](https://github.com/nf-core/tools/issues/370) to 4h
```

**Comment by apeltzer** [comment]
@nf-core/core Ready for review 👍

File: nf_core/pipeline-template/{{cookiecutter.name_noslash}}/conf/base.config [hunk 2]
```diff
@@ -14,8 +14,8 @@ process {
   // TODO nf-core: Check the defaults for all processes
   cpus = { check_max( 1 * task.attempt, 'cpus' ) }
   memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
-
+  time = { check_max( 4.h * task.attempt, 'time' ) } 
+
```
Comment (ewels) [inline nf_core/pipeline-template/{{cookiecutter.name_noslash}}/conf/base.config]: Whitespace 😱

**Comment by ewels** [comment]
7 commits for one character change 😆

**Comment by apeltzer** [comment]
Yeah, please stash if possible. Mainly because we had to merge in `dev` a couple of times...

**Pull request merged**